### PR TITLE
#205 Updated image search to use non deprecated api

### PR DESCRIPTION
**Pull request by shadow7412**
Fixed image me.
Please test this @tophsic and @antgel as I didn't get a chance.

### CODE REVIEW

**Comment by shadow7412** [comment]
@skoczen - looks like the tests aren't set up correctly? That failure looks like a lack of required environment variables to me..

File: will/plugins/productivity/images.py [hunk 1]
```diff
@@ -9,19 +10,29 @@ class ImagesPlugin(WillPlugin):
     @respond_to("image me (?P<search_query>.*)$")
     def image_me(self, message, search_query):
         """image me ___ : Search google images for ___, and post a random one."""
+
+        if not (getattr(settings, "GOOGLE_API_KEY", False) and getattr("GOOGLE_CUSTOM_SEARCH_KEY")):
```
Comment (jtriley) [inline will/plugins/productivity/images.py]: Think this should be 
```
getattr(settings, "GOOGLE_CUSTOM_SEARCH_KEY")
```

Comment (shadow7412) [inline will/plugins/productivity/images.py]: That means that this line will die with an AttributeError if the setting is not defined at all.

    class example(objects):
        real_attr = True

    getattr(example, real_attr)  # Returns True, as you'd expect
    getattr(example, fake_attr, False)  # Returns False
    getattr(example, fake_attr)  # Dies with AttributeError

However you have brought my attention to the second half of that line that will die in this exact way - so I'll fix that up.

**Comment by antgel** [comment]
What's the status of this PR? Would love to get this back.

**Comment by shadow7412** [comment]
As far as I'm aware it works... we've been running this code (or a variation thereof in a different plugin) for quite some time now. The image quality isn't as good (ie, it's not always on the mark) but it works.

Perhaps if you gave it a pull and made sure everything was working your end as well we might get some more momentum.

**Comment by skoczen** [comment]
Or, you know, if the maintainer wasn't so damn slow at responding and merging PRs ;)  

Thanks for your patience on this - and all the other PRs.  I've recently finished up at my last job, and in starting up a new company, odds are good that there will be time to work on will in the near future. 

Broadly, I liked this PR, but at first glance (which is all the closer I've looked),  I was a -1 to requiring a google key.   If there's a decent image search API we can use without asking people to set another config variable, I'd love that. 

Thanks again for hanging in there!

**Comment by skoczen** [comment]
Still would love to see an option that doesn't require a google key, but +1 to this being better than the current state!  Merging it in - thanks for all your work, @shadow7412 ! :)

**Comment by shadow7412** [comment]
I agree - but sadly I haven't come across a better solution. The other API was much better, shame google axed it :(

**Pull request merged**